projects/js-packages/api/index.jsx

```diff
@@ -162,6 +162,11 @@ function JetpackRestApiClient( root, nonce ) {
 				.then( checkStatus )
 				.then( parseJsonResponse ),
 
+		setHasSeenWCConnectionModal: () =>
+			postRequest( `${ apiRoot }jetpack/v4/seen-wc-connection-modal`, postParams )
```

Comment:
I initially thought about using `updateOptions` but after reading a bit the code and the context where it was used, it felt a bit out of place for what I wanted to do. In short, `updateOptions` calls the `v4/settings` API, and all the structure around it seems to be used for updating modules/general settings, not really flags which is the case here. 
But I'm open to hear your thoughts and change it if you think otherwise.

Reply:
A new endpoint just for Woo seems a bit overkill to me, but I would not consider this a blocker here. We can revisit this later.

Reply:
@oskosk What would you recommend here? Do you happen to remember a bit about `updateOptions`?

Thank you!